packages/fast-components-styles-msft/src/auto-suggest/index.ts

```diff
@@ -1,18 +1,17 @@
-import designSystemDefaults, {
-    DesignSystem,
-    withDesignSystemDefaults,
-} from "../design-system";
+import { DesignSystem, withDesignSystemDefaults } from "../design-system";
 import { ComponentStyles, ComponentStyleSheet } from "@microsoft/fast-jss-manager";
 import { AutoSuggestClassNameContract } from "@microsoft/fast-components-class-name-contracts-msft";
-import { elevation, ElevationMultiplier } from "../utilities/elevation";
 import { toPx } from "@microsoft/fast-jss-utilities";
+import { applyFloatingCornerRadius } from "../utilities/border";
 import { neutralFillStealthRest } from "../utilities/color";
-import { curry } from "lodash-es";
+import { heightNumber } from "../utilities/density";
```

Comment:
just a note, we really need to change this to be more descriptive

Reply:
Leaving as is per our conversation.